### PR TITLE
WIP PR for SRVKS-977 & SRVCOM-1884: Wrong container image used in serverless documentation example

### PR DESCRIPTION
**Pull request by kaldesai**
**Affected versions for cherry-picking:** serverless-docs 1.28+

**Dedicated JIRAs:** 
- https://issues.redhat.com/browse/SRVKS-1004
- https://issues.redhat.com/browse/SRVCOM-1884

**Description:** 

Made the changes in the following sections: 
- **Creating serverless applications using YAML**: Changed the YAML file 
- **Verifying your serverless application deployment**: Changed the verification steps 

**Doc preview:** 

- [Creating serverless applications using YAML](https://62713--docspreview.netlify.app/openshift-serverless/latest/knative-serving/getting-started/serverless-applications.html#creating-serverless-apps-yaml_serverless-applications)
- [Verifying your serverless application deployment](https://62713--docspreview.netlify.app/openshift-serverless/latest/knative-serving/getting-started/verifying-application-deployment.html#verifying-serverless-app-deployment_verifying-application-deployment)

**Reviews:** 
- SME: 
- QE: 
- Peer review:

### CODE REVIEW

**Comment by cardil** [comment]
Looks like the https://github.com/openshift/openshift-docs/pull/60772 is doing the same thing, doesn't it?

File: modules/verifying-serverless-app-deployment.adoc [hunk 1]
```diff
@@ -30,55 +30,55 @@ $ oc get ksvc <service_name>
 .Example output
 [source,terminal]
 ----
-NAME            URL                                        LATESTCREATED         LATESTREADY           READY   REASON
-event-delivery   http://event-delivery-default.example.com   event-delivery-4wsd2   event-delivery-4wsd2   True
+NAME       URL                                   LATESTCREATED    LATESTREADY      READY   REASON
+showcase   http://showcase-default.example.com   showcase-00001   showcase-00001   True
```
Comment (skonto) [inline modules/verifying-serverless-app-deployment.adoc:34]: Do you have a pointer of the draft doc for this? Is the output provided on jira?

**Comment by skonto** [comment]
LGTM. I did a quick search and don't see `event-delivery` coming up in other places. cc @cardil who is driving this for more.

**Comment by kaldesai** [comment]
This pull request is a duplicate of https://github.com/openshift/openshift-docs/pull/60772. Closing this PR

**Comment by kaldesai** [comment]
This pull request is a duplicate of https://github.com/openshift/openshift-docs/pull/60772.

**Pull request closed without merging**